src/olympia/amo/tests/__init__.py

```diff
@@ -717,11 +729,27 @@ def collection_factory(**kw):
     return c
 
 
+def license_factory(**kw):
+    data = {
+        'name': {
+            'en-US': u'My License',
+            'fr': u'Mä Licence',
+        },
+        'text': {
+            'en-US': u'Lorem ipsum dolor sit amet, has nemore patrioqué',
+        },
+        'url': 'http://license.example.com/',
+    }
+    data.update(**kw)
+    return License.objects.create(**data)
+
+
 def file_factory(**kw):
     v = kw['version']
+    filename = kw.pop('filename', '%s-%s' % (v.addon_id, v.id))
     status = kw.pop('status', amo.STATUS_PUBLIC)
     platform = kw.pop('platform', amo.PLATFORM_ALL.id)
-    f = File.objects.create(filename='%s-%s' % (v.addon_id, v.id),
+    f = File.objects.create(filename=filename,
```

Comment:
Please take the opportunity to rename the `f` and `v` variable :crying_cat_face: